.github/workflows/ci.yaml

```diff
@@ -5,6 +5,24 @@ on:
     pull_request:
 
 jobs:
+    php-cs-fixer:
+        runs-on: ubuntu-latest
+        name: Coding Standards
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v4
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+
+                with:
+                    php-version: 8.1
```

Comment:
Why not 8.3?

Reply:
I just copied the highest PHP version, didn't notice 8.2 & 8.3 are missing, fixed.